### ISSUE TITLE
WAI-ARIA 1.3 FPWD

### ISSUE THREAD

**Issue by daniel-montalvo** [w3ctag/design-reviews#927]
こんにちは TAG-さん!
                
I'm requesting a TAG review of Accessible Rich Internet Applications 1.3 FPWD.

WAI-ARIA, the Accessible Rich Internet Applications Suite, defines a way to make Web content and Web applications more accessible to people with disabilities. It especially helps with dynamic content and advanced user interface controls developed with HTML, JavaScript, and related technologies. Without WAI-ARIA certain functionality used in Web sites is not available to some users with disabilities, especially people who rely on screen readers and people who cannot use a mouse. WAI-ARIA addresses these accessibility challenges, for example, by defining ways for functionality to be provided to assistive technology. With WAI-ARIA, developers can make advanced Web applications accessible and usable to people with disabilities.

  - Explainer¹ (minimally containing user needs and example code): https://www.w3.org/WAI/standards-guidelines/aria/ (mostly focused on user needs)
  - Specification URL: https://www.w3.org/TR/wai-aria-1.3/
  - Tests: https://github.com/web-platform-tests/wpt/tree/master/wai-aria Available. Test are gradually added)
  - User research: none (minor spec update)
  - Security and Privacy self-review²: https://www.w3.org/TR/wai-aria-1.3/#security-considerations and https://www.w3.org/TR/wai-aria-1.3/#privacy-considerations
  - GitHub repo (if you prefer feedback filed there): https://www.github.com/w3c/aria/
  - Primary contacts (and their relationship to the specification):
      - James Nurthen, @jnurthen, Adobe
      - Valerie Young, @spectranaut, Egalia
      - Peter Krautzberger, @pkra, krautzource UG 
  - Organization(s)/project(s) driving the specification: WAI-ARIA (W3C, Adobe, Apple, Google, Mozilla, Meta, krautzource UG )
  - Key pieces of existing multi-stakeholder (e.g. developers, implementers, civil society) support, review or discussion of this specification:
    - Chromium comments: NA
    - Mozilla comments: NA
    - WebKit comments: NA
  - External status/issue trackers for this specification (publicly visible, e.g. Chrome Status): NA

Further details:

  - [x] I have reviewed the TAG's [Web Platform Design Principles](https://www.w3.org/TR/design-principles/)
  - Relevant time constraints or deadlines: Ideally by 1st March 2024. This is a minor spec update with some [changes](https://www.w3.org/TR/wai-aria-1.3/Echangelog) with respec to  WAI-ARIA 1.2
  - The group where the work on this specification is currently being done: ARIA Working Group
  - The group where standardization of this work is intended to be done (if current group is a community group or other incubation venue): NA
  - Major unresolved issues with or opposition to this specification: NA
  - This work is being funded by: orgs mentioned above

You should also know that...

[please tell us anything you think is relevant to this review]

We'd prefer the TAG provide feedback as (please delete all but the desired option):

  🐛 open issues in our GitHub repo for **each point of feedback**

------------------------------------------------------------------------------------
CAREFULLY READ AND DELETE CONTENT BELOW THIS LINE BEFORE SUBMITTING

Please preview the issue and check that the links work before submitting.

In particular, if anything links to a URL which requires authentication (e.g. Google document), please make sure anyone with the link can access the document. We would prefer fully public documents though, since we work in the open.

¹ We require an explainer to give the relevant context for the spec review, even if the spec has some background information. For background, see our [explanation of how to write a good explainer](https://tag.w3.org/explainers/). We recommend the explainer to be in [Markdown](https://github.github.com/gfm/).

² A Security and Privacy questionnaire helps us understand potential security and privacy issues and mitigations for your design, and can save us asking redundant questions. See https://www.w3.org/TR/security-privacy-questionnaire/.

³ For your own organization, you can simply state the organization's position instead of linking to it. Chromium doesn't have a standards-positions repository and [prefers](https://source.chromium.org/chromium/chromium/src/+/main:docs/standards/positions/GoogleChrome/README.md) to use comments from the teams that maintain the relevant area of their codebase.

**Comment by matatk** [comment]
We were just discussing this review in a TAG breakout. Here's a link to the [significant changes since ARIA 1.2](https://www.w3.org/TR/wai-aria-1.3/#changelog).

**Comment by matatk** [comment]
Thanks for your review request. We have no architectural concerns. There are a lot of really helpful new features, changes, and clarifications in this version, which will be of great help to users; great work!
